cedar-policy-validator/src/typecheck/test/expr.rs

```diff
@@ -762,29 +798,26 @@ fn in_set_typechecks_strict() {
 #[test]
 fn in_typecheck_fails() {
     let src = "0 in true";
-    assert_typecheck_fails_empty_schema(
-        src.parse().unwrap(),
-        Type::primitive_boolean(),
+    let errors =
+        assert_typecheck_fails_empty_schema(src.parse().unwrap(), Type::primitive_boolean());
+    assert_eq!(errors.len(), 2);
+    assert!(errors.contains(&ValidationError::expected_type(
```

Comment:
I think assertion failures will be substantially worse here. The old assertion would have done an `assert_eq!` on the error sets, printing a pretty diff because of the `similar-asserts` dependency while this assert just fails.

Reply:
Good point. (also I forgot about `similar-asserts`, thanks for the reminder)

Reply:
Fixed in latest revision